Tasks/Maven/maventask.ts

```diff
@@ -212,11 +223,55 @@ function createMavenSQRunner(sqHostUrl, sqHostUsername, sqHostPassword, sqDbUrl?
 function applyPmdGoals(mvnRun: trm.ToolRunner):void {
     if (tl.getInput('pmdAnalysisEnabled', true) != 'true') {
         console.log('PMD analysis is not enabled');
+        return;
     }
     console.log('PMD analysis is enabled');
     pmd.applyPmdArgs(mvnb);
 }
 
+// Extract data from code analysis output files and upload results to build server
+function postCodeAnalysisResults():void {
+    // Need to run this if any one of the code analysis tools were run
+    if (tl.getInput('pmdAnalysisEnabled', true) == 'true') {
```

Comment:
This line will inflate with OR statements as we add more tools, good point.

Accepted.